crates/turbopack-node/src/render/mod.rs

```diff
@@ -19,6 +20,7 @@ pub struct RenderData {
     raw_query: String,
     raw_headers: Vec<(String, String)>,
     path: String,
+    data: Option<JsonValue>,
```

Comment:
```suggestion
    data: Option<turbo_tasks::JsonValueReadRef>,
```

That should allow to serialize it without cloning

Reply:
Done, but this required me to update the `Deserialize` impl for `ReadRef<T>` to `ReadRef<T, U>`. Please verify if I did that correctly.